### ISSUE TITLE
Unable to run Demo app

### ISSUE THREAD

**Issue by schwarcu** [zostera/django-bootstrap4#84]
`PS C:\Users\user\Downloads\django-bootstrap4-master\django-bootstrap4-master\demo> python manage.py runserver
Unhandled exception in thread started by <function check_errors.<locals>.wrapper at 0x060BFE88>
Traceback (most recent call last):
  File "C:\Users\user\AppData\Local\Programs\Python\Python36-32\lib\site-packages\django\utils\autoreload.py", line 225, in wrapper
    fn(*args, **kwargs)
  File "C:\Users\user\AppData\Local\Programs\Python\Python36-32\lib\site-packages\django\core\management\commands\runserver.py", line 112, in inner_run
    autoreload.raise_last_exception()
  File "C:\Users\user\AppData\Local\Programs\Python\Python36-32\lib\site-packages\django\utils\autoreload.py", line 248, in raise_last_exception
    raise _exception[1]
  File "C:\Users\user\AppData\Local\Programs\Python\Python36-32\lib\site-packages\django\core\management\__init__.py", line 327, in execute
    autoreload.check_errors(django.setup)()
  File "C:\Users\user\AppData\Local\Programs\Python\Python36-32\lib\site-packages\django\utils\autoreload.py", line 225, in wrapper
    fn(*args, **kwargs)
  File "C:\Users\user\AppData\Local\Programs\Python\Python36-32\lib\site-packages\django\__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "C:\Users\user\AppData\Local\Programs\Python\Python36-32\lib\site-packages\django\apps\registry.py", line 89, in populate
    app_config = AppConfig.create(entry)
  File "C:\Users\user\AppData\Local\Programs\Python\Python36-32\lib\site-packages\django\apps\config.py", line 90, in create
    module = import_module(entry)
  File "C:\Users\user\AppData\Local\Programs\Python\Python36-32\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 953, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'tests'`

**Comment by serajushsalekin** [comment]
i think in your settings.py you didn't entry your app name in installed app

**Comment by amard33p** [comment]
I got the same error. However even if I do 'pip install tests' I run into another error:
```
  File "H:\django-bootstrap4\demo\forms.py", line 7, in <module>
    from tests.test_templatetags import TestForm
ModuleNotFoundError: No module named 'tests.test_templatetags'
```

**Comment by dlparker** [comment]
One of us is missing something. If I clone the repo, setup a virtualenv, install the code, install the requirements, and then try to run the demo, I get the same error reported above. That doesn't have anything to do with me adding my own app to anything. If there is a need to do that, it isn't documented anywhere.

**Comment by Sidon** [comment]
When I try to run the demo, I get the error:

File "/path/demo/demo/forms.py", line 7, in <module>
    from tests.test_templatetags import TestForm
ModuleNotFoundError: No module named 'tests.test_templatetags'

**Comment by dyve** [comment]
The goal is to have the demo app run when you clone the repo and run `python manage.py runserver`. Currently, this does not work. This needs to be fixed.

**Comment by bvishwajit** [comment]
Same problem here. Has anyone found a solution yet?

**Comment by dyve** [comment]
Working on it.

**Comment by dyve** [comment]
Fixed in 921861d3db0628fa2a174e02312f8ac17e2dd500 (although the demo app itself still has an error from the bootstrap3 days)
